app/src/main/java/com/seay/game_of_thrones/inject/AppComponent.java

```diff
@@ -15,6 +17,15 @@
     //Activities
     void inject(MainActivity activity);
 
+    //AndroidViewModels
+    void inject(CharacterAndroidViewModel viewModel);
+
     //Models
     void inject(Welcome welcome);
+
+    //Jobs
+    void inject(GetCharacterListJob job);
+
+    //Repositories
+    void inject(CharacterRepository characterRepository);
```

Comment:
I'm being super nit-pickey, but can we change `characterRepository` to `repository`? This will make adding more easier.